front/migrations/db/migration_124.sql

```diff
@@ -0,0 +1,11 @@
+-- Drop existing foreign key constraint
+ALTER TABLE retrieval_documents
+DROP CONSTRAINT "retrieval_documents_retrievalActionid_fkey";
```

Comment:
Still not the right name, in production we have: `retrieval_documents_retrievalActionId_fkey`.

Reply:
My fault, I copied from my local. Copied it from prodbox, should be fine now 🙏🏼